### ISSUE TITLE
Not working on Laravel 9

### ISSUE THREAD

**Issue by hosmelq** [laravel/pulse#95]
### Pulse Version

beta

### Laravel Version

v9.52.16

### PHP Version

8.2

### Description

Pulse is not working since `Benchmark::value` was added in Laravel 10.

I can send a PR, but I want to know the best way. Maybe add the method in Laravel 9, add the `value` method directly to the `RemembersQueries` trait, or inline the code in the `remember` method.

### Steps To Reproduce

Install Pulse in a Laravel 9 project.

**Comment by devajmeireles** [comment]
I suspect Pulse was designed for Laravel 10+ only.

**Comment by dlopez525** [comment]
@hosmelq What error did the sale give you? blank page?

**Comment by dlopez525** [comment]
@devajmeireles I installed it in Laravel 10, but it doesn't work, I have been updating that app from Laravel 5.8. But in an app installed from Laravel 10, it works.

**Comment by hosmelq** [comment]
> I suspect Pulse was designed for Laravel 10+ only.

It targets Laravel 9

https://github.com/laravel/pulse/blob/3afeb3f59af17e6d496ac77f7b8bdd38384d8902/composer.json#L21

**Comment by dlopez525** [comment]
@hosmelq Could you explain what you did?

**Comment by crynobone** [comment]
I believe in was an oversight to add Laravel 9 support. Pulse require Livewire 3 which obly support Laravel 10.

**Comment by dlopez525** [comment]
@crynobone I have an app in Laravel 10 and it doesn't work

**Comment by hosmelq** [comment]
> I believe in was an oversight to add Laravel 9 support. Pulse require Livewire 3 which only support Laravel 10.

I agree with you. I'm sure they will remove the number 9 from the composer.json file.

**Comment by te1** [comment]
List of reasons why it doesn't work with Laravel 9 out of the box (that I found so far):
- Needs nunomaduro/collision 7.7 or newer (Laravel 9 uses ^6.1)
- Needs illuminate/support 10 (not enforced by composer.json right now)
  - Dashboard needs `Illuminate\Support\Benchmark::value`
  - `php artisan pulse:check` needs `Illuminate\Support\Sleep`

If you manually patch that it seems to work in Laravel 9. Still probably best to just enforce Laravel 10 though.
